docs/src/howto-guides/019_how_to_use_preferred_service_manager.md

```diff
@@ -0,0 +1,49 @@
+# How to use thin-edge.io with your preferred init system
+
+thin-edge.io works seamlessly with `systemd` on the CLI commands `tedge connect` and `tedge disconnect`.
+However, not all OS support `systemd`.
+You might want to use other init systems like `OpenRC`, `BSD`, `init.d` with thin-edge.io.
+For this demand, this guide explains how to give a custom init system configuration to thin-edge.io.
+
+## How to set a custom init system configuration
+
+Create a file `system.toml` owned by `root:root` in `/etc/tedge` directory.
+
+```shell
+sudo touch /etc/tedge/system.toml
+```
+
+Open your editor and copy and paste the following toml contents.
+This is an example how to configure OpenRC as the init system for thin-edge.io.
+We have example configurations for BSD, OpenRC, and systemd under [thin-edge.io/configuration/contrib/system](https://github.com/thin-edge/thin-edge.io/tree/main/configuration/contrib/system).
+
+
+```toml
+[init]
+name = "OpenRC"
+is_available = ["/sbin/rc-service", "-l"]
+restart = ["/sbin/rc-service", "{}", "restart"]
+stop =  ["/sbin/rc-service", "{}", "stop"]
+enable =  ["/sbin/rc-update", "add", "{}"]
+disable =  ["/sbin/rc-update", "delete", "{}"]
+is_active = ["/sbin/rc-service", "{}", "status"]
+```
+
+Then, adjust the values with your preferred init system.
+To get to know the rules of the configuration file, please refer to [Init System Configuration File](./../references/system-config.md).
```

Comment:
Probably we should also update that linked files name to "init-system-config.md". That way its more clear that it is only about the init system.

Reply:
I think you changed the file name, and my PR used the old name :D